Remove deprecated do clause which is not supported by perl 5 any more.

Signed-off-by: Kai Kang <kai.kang@windriver.com>

--- racoon2-20100526a/pskgen/pskgen.in.orig	2014-01-27 16:19:19.028054860 +0800
+++ racoon2-20100526a/pskgen/pskgen.in	2014-01-27 16:19:47.707748768 +0800
@@ -59,8 +59,8 @@
 	exit 0;
 }
 
-require 'getopts.pl';
-do Getopts('rs:o:di:he:d');
+use Getopt::Std;
+getopts('rs:o:di:he:d');
 $output = '-';
 $output = $opt_o if ($opt_o);
 
